pkgs/top-level/all-packages.nix

```
@@ -17498,6 +17498,14 @@ in
 
   librealsense = callPackage ../development/libraries/librealsense { };
 
+  librealsenseWithCuda = callPackage ../development/libraries/librealsense {
+    cudaSupport = true;
+  };
+
+  librealsenseWithoutCuda = callPackage ../development/libraries/librealsense {
```

I believe the current convention is just to have no suffix
```suggestion
  librealsense = callPackage ../development/libraries/librealsense {
```

---

the "default" version uses config to switch, whereas this one forces to always "not" use cuda